File: app/src/main/java/com/moxun/tagcloud/FragmentTestActivity.java
package com.moxun.tagcloud;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class FragmentTestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fragment_test);
        getSupportFragmentManager()
                .beginTransaction()
                .add(R.id.container, TestFragment.newInstance())
                .commit();
    }
}

File: app/src/main/java/com/moxun/tagcloud/FragmentTestActivity.kt
package com.moxun.tagcloud

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.moxun.tagcloud.TestFragment

class FragmentTestActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment_test)
        supportFragmentManager.beginTransaction()
                .add(R.id.container, TestFragment.Companion.newInstance())
                .commit()
    }
}